docs/integrations/sources/shopify.inapp.md

```diff
@@ -1,30 +1,26 @@
 ## Prerequisites
 
-* An Active [Shopify store](https://www.shopify.com) 
-* The Admin API access token of your [Custom App](https://help.shopify.com/en/manual/apps/app-types/custom-apps).
+* An Active [Shopify store](https://www.shopify.com)
 
-:::note
+::: note
+
+If you are already a Cloud user but still use the `API PASSWORD` authentication method (option), please make sure you've switched to the `OAuth2.0`, as far as the `API PASSWORD` is going to be deprecated soon for Cloud Users, but this option remains for `OSS` users.
```

Comment:
Suggestion: If you previously used the `API Password` authentication method, please switch to the `OAuth2.0`, as the `API Password` will be deprecated shortly. If you prefer to continue using `API Password`, the option will remain available in Airbyte Open-Source.